modules/140-user-authz/templates/cluster-roles.yaml

```
@@ -324,7 +324,7 @@ metadata:
   name: user-authz:cluster-editor
   {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
 rules:
-{{- include "user_authz_common_rules" "ClusterAdmin" }}
+{{- include "user_authz_common_rules" "ClusterEditor" }}
```

Obviously, it is some garbage.
```suggestion
{{- include "user_authz_common_rules" "ClusterAdmin" }}
```

---

removed